pom.xml

```diff
@@ -1,139 +1,149 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- $Id: pom.xml 39481 2011-05-12 20:14:54Z kutzi $ -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>plugin</artifactId>
-        <version>4.12</version>
-    </parent>
-    <groupId>org.jvnet.hudson.plugins</groupId>
-    <artifactId>ircbot</artifactId>
-    <packaging>hpi</packaging>
-    <version>${revision}${changelist}</version>
-    <name>Jenkins IRC Plugin</name>
-    <description>A build status publisher that notifies channels on a IRC server</description>
-    <url>https://github.com/jenkinsci/ircbot-plugin</url>
-    <licenses>
-        <license>
-            <name>MIT license</name>
-            <comments>All source code is under the MIT license.</comments>
-        </license>
-    </licenses>
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>4.13</version>
+  </parent>
+  <groupId>org.jvnet.hudson.plugins</groupId>
+  <artifactId>ircbot</artifactId>
+  <packaging>hpi</packaging>
+  <version>${revision}${changelist}</version>
+  <name>Jenkins IRC Plugin</name>
+  <description>A build status publisher that notifies channels on a IRC server</description>
+  <url>https://github.com/jenkinsci/ircbot-plugin</url>
+  <licenses>
+    <license>
+        <name>MIT license</name>
+        <comments>All source code is under the MIT license.</comments>
+    </license>
+  </licenses>
 
-    <properties>
-        <revision>2.36</revision>
-        <changelist>-SNAPSHOT</changelist>
-        <gitHubRepo>jenkinsci/ircbot-plugin</gitHubRepo>
-        <version.instant-messaging.plugin>1.39</version.instant-messaging.plugin>
-        <java.level>8</java.level>
-        <jenkins.version>2.269</jenkins.version>
-    </properties>
-
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.204.x</artifactId>
-                <version>17</version>
-                <scope>import</scope>
-                <type>pom</type>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
+  <properties>
+    <revision>2.34</revision>
```

Comment:
Probably a mis-merge from fixing over an older master commit? :) Probably related to the different white-spacing applied to the file here and in master.

Reply:
Fixed